modules/cli/src/main/scala/scala/cli/launcher/LauncherCLI.scala

```diff
@@ -0,0 +1,65 @@
+package scala.cli.launcher
+
+import caseapp.core.RemainingArgs
+import dependency._
+
+import scala.build.internal.Runner
+import scala.build.options.BuildOptions
+import scala.build.{Artifacts, Positioned}
+import scala.cli.commands.{LoggingOptions, ScalaCommand}
+
+object LauncherCLI extends ScalaCommand[LauncherOptions] {
```

Comment:
We probably don't need this to be a `ScalaCommand` or a case-app thing. A simple method, like `def runAndExit(version: String, args: Seq[String]): Nothing` should work (see my comment above).